docs-api/frontity-packages/features-packages/wp-comments.md

```diff
@@ -0,0 +1,226 @@
+---
+description: API reference of `@frontity/wp-comments` package
+---
+
+# `@frontity/wp-comments`
+
+Comments package that adds integration for WordPress native comments.
+
+<!-- toc -->
+
+- [Installation](#installation)
+- [Settings](#settings)
+  - [In WordPress](#in-wordpress)
+  - [In Frontity](#in-frontity)
+- [API Reference](#api-reference)
+  - [Handlers](#handlers)
+    - [`@comments/:id`](#commentsid)
+  - [State](#state)
+    - [`state.comments.forms[]`](#state-comments-forms)
+  - [Actions](#actions)
+    - [`actions.comments.updateFields()`](#actions-comments-updatefields)
+    - [`actions.comments.submit()`](#actions-comments-submit)
+- [Demo](#demo)
+
+<!-- tocstop -->
+
+## Installation
+
+Add the `wp-comments` package to your project:
+
+```bash
+npm i @frontity/wp-comments
+```
+
+## Settings
+
+### In WordPress
+
+In order to use this package, you will need to add a single line of configuration to your Wordpress installation:
+
+```php
+add_filter( 'rest_allow_anonymous_comments', '__return_true' );
+```
+
+You can add that directly in your `theme.php` file or use a [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin.
```

Comment:
Replace
"You can add that directly in your `theme.php` file..."
with
"You can add that directly in your theme's `functions.php` file..."